src/main/java/org/terasology/lost/generator/LostWorldGenerator.java

```diff
@@ -117,26 +119,34 @@ public Vector3f getSpawnPosition(EntityRef entity) {
             BiomeModel biomeModel = model.get(g);
             for (org.terasology.polyworld.graph.Region r : g.getRegions()) {
                 WhittakerBiome biome = biomeModel.getBiome(r);
-                boolean flag = false;
-                boolean flag2 = false;
+                boolean ocean = false;
+                boolean forest = false;
+                boolean desert = false;
                 if(biomeModel.getBiome(r).equals(WhittakerBiome.TROPICAL_SEASONAL_FOREST)||biomeModel.getBiome(r).equals(WhittakerBiome.TEMPERATE_RAIN_FOREST)||biomeModel.getBiome(r).equals(WhittakerBiome.TEMPERATE_DECIDUOUS_FOREST)||biomeModel.getBiome(r).equals(WhittakerBiome.TROPICAL_RAIN_FOREST)){
```

Comment:
Please extract the biome check from
```if(biomeModel.getBiome(r).equals(WhittakerBiome.TROPICAL_SEASONAL_FOREST)||biomeModel.getBiome(r).equals(WhittakerBiome.TEMPERATE_RAIN_FOREST)||biomeModel.getBiome(r).equals(WhittakerBiome.TEMPERATE_DECIDUOUS_FOREST)||biomeModel.getBiome(r).equals(WhittakerBiome.TROPICAL_RAIN_FOREST)){
```

into something like `boolean isForestBiome(WhittakerBiome biome)`

Same for the checks for the other biomes.